### ISSUE TITLE
[Feature Suggestion] Integration with VIM Easy Motion

### ISSUE THREAD

**Issue by mrvicadai** [onivim/oni2#517]
As a heavy vim user, one of the essential plugin is easymotion: https://github.com/easymotion/vim-easymotion

This will allow user jump to arbitrary point on the screen without having to navigate to line first, then to a word, e.g. `123G5w`.

**Comment by yunti** [comment]
Looking forward to this and in particular vim-surround when vim plugin support comes.

**Comment by Raikiri** [comment]
Yes, please. Unfortunately, easymotion does look kinda like something that Oni2 might not necessarily support from vim plugins as it does change coloring of text which might not play nicely with Oni2's rendering engine. There are a couple of vs code plugins that do things similarly, but in my opinion modern easymotion is still the best.

**Comment by JoeSchr** [comment]
[vscode-neovim](https://marketplace.visualstudio.com/items?itemName=asvetliakov.vscode-neovim) made a custom plugin [fork ](https://github.com/asvetliakov/vim-easymotion )to use vim-easymotion with vscode. it's pretty great and a essential for me. 

I can't use vim without this anymore... 

so my oni will not be open much until it supports this kind of jumping around

**Comment by vaheqelyan** [comment]
that would be great. however I use https://github.com/justinmk/vim-sneak

**Comment by Djo1e** [comment]
One of my favorite plugins - would love to see it happen! +1

**Comment by vdwees** [comment]
This feature is so useful that it is baked directly into editors like [amp](https://amp.rs/)
